doc/text/release.md

```diff
@@ -74,18 +74,22 @@ Then, edit PO files:
 
 ## 3. Upload gem to RubyGems.org
 
-### 3-1. For Linux
+### 3-1. Bump version if need
 
-    $ rake release
+Change `RB_GRN_MAJOR_VERSION`, `RB_GRN_MINOR_VERSION` and `RB_GRN_MICRO_VERSION`
+to the same version of the latest Groonga.
+
+    $ $EDITOR ext/groonga/rb-grn.h
 
-### 3-2. For Windows
+### 3-2. Release
 
-    $ rake build:windows
-    $ gem push pkg/rroonga-<the latest release version>-x86-mingw32.gem
-    $ gem push pkg/rroonga-<the latest release version>-x64-mingw32.gem
+    $ rake release
 
 ### 3-3. Bump version
 
+Increment `RB_GRN_MICRO_VERSION`.
+If current `RB_GRN_MICRO_VERSION` is `9`, increment `RB_GRN_MINOR_VERSION` and reset `RB_GRN_MICRO_VERSION` to `0`.
```

Comment:
@yoshimotoyuk Do we need "the" before "current"?

Reply:
I don't think it is necessary because  `RB_GRN_MICRO_VERSION` seems individual general noun, that direct specific thing. 
But if this is talking about `RB_GRN_MICRO_VERSION` in specific situation , it would be better to have"the" to differentiate the word.

Reply:
Thanks!